Servers/Administration/OJS.Servers.Administration.Infrastructure/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -18,10 +20,28 @@ public static void ConfigureServices<TProgram>(this IServiceCollection services)
             => services
                 .AddWebServer<TProgram>()
                 .AddHangfireServer(AppName)
+                .ValidateLaunchSettings()
                 .AddIdentityDatabase<AdminDbContext, UserProfile, Role, UserInRole>(Enumerable.Empty<GlobalQueryFilterType>())
                 .AddScoped<OjsDbContext, AdminDbContext>()
                 .AddSoftUniJudgeCommonServices()
                 .UseAutoCrudAdmin()
                 .AddControllersWithViews();
+
+        private static IServiceCollection ValidateLaunchSettings(this IServiceCollection services)
```

Comment:
Extract into common method and reuse in both Admin and Ui. Also make the method accept params, so that in Admin can be called like this `.ValidateLaunchSettings(EnvironmentVariables.LocalTimeZone)` the rest of the env variables can go into the method, like they are currently, as they are required for all apps.